crocodealer.cabal

```
@@ -23,6 +23,7 @@ source-repository head
 library
   hs-source-dirs:      src
   exposed-modules:     Crocodealer
+                       Crocodealer.Config
```

It's not necessary required, but I prefer to indent hierarchical modules in the `.cabal` file as well for readability. See `issue-wanted` for example:

* https://github.com/kowainik/issue-wanted/blob/e612a87454b207fa47f64c630b17d014cdc4aa8a/issue-wanted.cabal#L61-L81

```suggestion
                           Crocodealer.Config
```